### ISSUE TITLE
Black Screen

### ISSUE THREAD

**Issue by Suidddz** [dnschneid/crouton#3065]
```
I honestly don't know how to paste this
```

#### Please describe your issue:
After going through all the steps to install crouton, when I type "start xfce4" , the screen goes black and stays that way. there's no mouse cursor, nothing. Is there any way to fix this?

#### If known, describe the steps to reproduce the issue:
not sure

**Comment by MCBGamingYT** [comment]
i get it too except my secondary screen shows No Signal

**Comment by haojiezhu** [comment]
Happens here too on an ASUS Chromebox with Trusty & Xfce. This seems to happen to many people after the latest ChromeOS update (56.0.2924.87). We'll just have to wait for a patch.

**Comment by MegaWalls** [comment]
This happened to me too. I am sure that this is has to do with the chrome OS update, I am waiting for a crouton patch, but what should I do when the patch comes out? Do I need to use a command to update it? Or will it update automatically?

**Comment by MegaWalls** [comment]
just used the update command (sudo sh ~/Downloads/crouton -u -n chrootname). 
It doesn't work.

**Comment by DennisLfromGA** [comment]
https://github.com/dnschneid/crouton/issues/3057#issuecomment-279196855

**Comment by dpilgrim** [comment]
I've been battling this issue for the last 6 hours. I advise patience. In my case I have a Toshiba Chromebook 2, and all my problems started after the latest Chrome OS update. I'm now running:
Version 56.0.2924.87 (64-bit)
Platform 9000.82.0 (Official Build) stable-channel swanky
Firmware Google_Swanky.5216.238.5

First I got the black screen. At this point I was running trusty. Initially I could tab (ctrl-alt-shift-previous) to Chrome OS and kill my terminal from there. Then I updated crouton. Still the black screen, but now no way to get back to Chrome OS. So I deleted my chroot, did a complete crouton reinstall. Now the reinstall failed with Xorg error messages. Next I deleted my chroot and went to a fresh install targeting xenial. Install failed with the same Xorg error messages. Next I deleted my chroot, did a CLI target install of xenial. This at least let me get back into a crouton session. Just no GUI. From there I tried to install Xorg from within the CLI. Installed fine, but any attempt to start a graphical session (startx or xinit) crashed. 

Then I noted that 2 hours ago, dnschneid had committed a new build of crouton with the brilliant but cryptic comment "omg. Everything is on fire." So I did a crouton upgrade, tried the xenial X11 install again, and it worked! Except that now I get the black screen.

So clearly the Version 56 build of Chrome OS has borked something with modelines in Xorg for Crouton. Not surprising. I still have nightmares about shot-in-the-dark modeline configs from late 1990s versions of Linux with sketchy graphics card support. I expect it will all get sorted out, but it isn't going to be an instant fix.

For what it's worth, dnshneid is a hero. He's given us a transformational piece of software that is a game changer for our Chrome devices, and gotten exactly the thanks that most open source developers get: little thanks, and many complaints from users who are incapable of making source code contributions themselves (myself included). I would pay good money for crouton if there were a way to do so. In the mean time, I'll patiently wait for the patch that fixes all this.
